### PR TITLE
Bard: Add Chat feature

### PR DESCRIPTION
**Pull request by EvanZhouDev**
## Description
Added the AI Chat command, allowing continued conversation with Bard.
- Ability to remember what you've said
- Start multiple "threads" with Bard

## Screencast
raycast-bard-ai-4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/raycast-bard-ai/TODO.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1. arguments
```
[the remark this answers is not in the document]
Comment (EvanZhouDev) [reply extensions/raycast-bard-ai/TODO.md:1]: Yeah, just fixed it...

File: extensions/raycast-bard-ai/src/api/common.jsx [hunk 2]
```diff
@@ -122,6 +137,7 @@ export default function ResultView(prompt, toast_title, type = "text", title, op
         !loading && (
           <ActionPanel title="Actions">
             <Action.CopyToClipboard title="Copy Results" content={pasteContent} />
+            {/* <Action title="Open in Chat" onAction={() => openInChat(response)} shortcut={{ modifiers: ["cmd", "shift"], key: "k" }} icon={Icon.Message} /> */}
```
Comment (sxn) [inline extensions/raycast-bard-ai/src/api/common.jsx:140]: should this be commented?

Comment (EvanZhouDev) [reply extensions/raycast-bard-ai/src/api/common.jsx:140]: Chat is still a work in progress. Google changed their API, breaking my code, so I am pushing this mainly to fix that...

Comment (EvanZhouDev) [reply extensions/raycast-bard-ai/src/api/common.jsx:140]: Well I mean the only thing that needs to be done is bump `bard-ai` version to v1.4.2, so what do you suggest? My internet is really slow right now, so making remote git changes is very tedious...

Comment (sxn) [reply extensions/raycast-bard-ai/src/api/common.jsx:140]: I'd suggest marking this PR as draft, and I can open another one that updates Bard to the version that gets the extension to work again. Then you have a bit of time to implement the chat without stressing out about the extension being broken. :)

Comment (sxn) [reply extensions/raycast-bard-ai/src/api/common.jsx:140]: https://github.com/raycast/extensions/pull/7320

File: extensions/raycast-bard-ai/TODO.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sxn) [reply extensions/raycast-bard-ai/TODO.md:1]: seems like the file's still here 😬

**Comment by sxn** [comment]
hey, thanks for taking the time to finally implement this! The overall change seems fine, the only problem is I can't get the extension as a whole to work 🤔 any idea what the problem could be?

https://github.com/raycast/extensions/assets/1155589/a2826bb5-c6ac-4e66-855c-af1957c2af20

**Comment by sxn** [comment]
The feature implemented in the PR looks good. I couldn't get Bard to work, but that's the case for the store version as well… 

Let's merge it. 🚀

**Comment by immortalsantee** [comment]
I tried to add `__Secure-1PSID cookie`.  Here is the value i copied.
image

After adding the value to `Google Bard Key`, I see this.
image

What is wrong. Please let me know.

Thanks

**Pull request merged**